tests/arrows/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -326,11 +326,11 @@ const middleware = options => (req, res, next) => {
   // ...
 };
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-const fn = b => c => d => {
+const fn = b => c => (d) => {
```

Comment:
this doesn't seem great in the currying case, but special-casing would certainly be awful